core/sdk-core/src/main/java/software/amazon/awssdk/core/client/config/ClientOverrideConfiguration.java

```diff
@@ -263,49 +260,6 @@ public String toString() {
                        .build();
     }
 
-    /**
-     * Adds the overridden values to the client configuration builder.
-     */
-    public SdkClientConfiguration.Builder addOverridesToConfiguration(SdkClientConfiguration.Builder builder) {
```

Comment:
Moved (again) to a different util class to avoid the direct coupling between `ClientOverrideConfiguration` which is an "public" class and the `SdkClientConfiguration` which is an "internal" class.